core/src/main/java/io/ap4k/project/SbtInfoReader.java

```diff
@@ -13,16 +13,51 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  * 
-**/
+ **/
 
 package io.ap4k.project;
 
 import java.nio.file.Path;
+import java.nio.file.Files;
+import java.util.HashMap;
+import java.util.Map;
+import java.io.IOException;
+import io.ap4k.Ap4kException;
+import java.io.InputStreamReader;
+import java.io.BufferedReader;
+import java.util.stream.Collectors;
+import java.util.Arrays;
 
 public class SbtInfoReader implements BuildInfoReader {
```

Comment:
As you added `sbt` tool for Scala language, does that mean that `ap4k` annotations will be supported for Scala projects ? Is it documented ?